enduro.toml

```diff
@@ -83,3 +83,9 @@ disabled = true
 
 [metadata]
 processNameMetadata = false
+
+[worker]
+heartbeatThrottleInterval = "5m"
+
+[activity]
+heartbeatTimeout = "1h"
```

Comment:
We should use the actual defaults when possible and change them in their env config.

Reply:
Done. f936fa7df08e26e9a20df1403f315a2ff8b32fce